readme.md

```diff
@@ -1,20 +1,109 @@
 <p align="center">
-  <img src="https://docs.ai-jsx.com/img/ai-jsx.png" alt="AI.JSX logo" width="300" />
+  <img src="https://docs.ai-jsx.com/img/ai-jsx.png" alt="AI.JSX logo" width="100" />
 </p>
 
-# AI.JSX
+# AI.JSX — The AI Application Framework for Javascript
 
-## Getting Started
+[![Docs Site](https://img.shields.io/badge/Docs%20Site-docs.ai--jsx.com-orange)](https://docs.ai-jsx.com)
+[![Discord Follow](https://dcbadge.vercel.app/api/server/MsKAeKF8kU?style=flat)](https://discord.gg/MsKAeKF8kU)
+[![Twitter Follow](https://img.shields.io/twitter/follow/fixieai?style=social)](https://twitter.com/fixieai)
 
-1. **Install [Volta](https://volta.sh/).** This manages the version of global tools like NodeJS and yarn.
-1. Set env var `OPENAI_API_KEY`.
+AI.JSX is a framework for building AI applications using [JSX](https://react.dev/learn/writing-markup-with-jsx). With AI.JSX, you don't just use JSX to describe what your UI should look like, you also use it to describe how **AI (LLMS)** should integrate into the rest of your application. The end result is a powerful combination where _intelligence_ can be deeply embedded into the application stack.
 
-Optional:
+AI.JSX is designed to give you two important capabilities out of the box:
 
-- To run the GitHub NextJS demo: set env var `GITHUB_TOKEN`.
+1. A powerful mechanism for orchestrating multiple LLM calls expressed as modular, re-usable components.
+1. The ability to seamlessly interweave UI components with your AI compoents. This means you can rely on the LLM to construct your UI dynamically from a set of standard React components.
```

Comment:
```suggestion
1. The ability to seamlessly interweave UI components with your AI compoents. This means you can rely on the LLM to construct your UI dynamically from a set of React components you provide.
```

"Standard" makes it sound like "there's a standard library", rather than "well-known components you provide".